pcmdi_metrics/graphics/parallel_coordinate_plot/parallel_coordinate_plot_mean_clim.ipynb

```diff
@@ -208,7 +208,7 @@
       "var_list: ['pr', 'prw', 'psl', 'rlds', 'rltcre', 'rlus', 'rlut', 'rlutcs', 'rsds', 'rsdscs', 'rsdt', 'rstcre', 'rsut', 'rsutcs', 'sfcWind', 'ta-200', 'ta-850', 'tas', 'tauu', 'ts', 'ua-200', 'ua-850', 'va-200', 'va-850', 'zg-500']\n",
       "var_unit_list: ['pr [N/A]', 'prw [N/A]', 'psl [N/A]', 'rlds [N/A]', 'rltcre [W m-2]', 'rlus [N/A]', 'rlut [N/A]', 'rlutcs [N/A]', 'rsds [N/A]', 'rsdscs [N/A]', 'rsdt [N/A]', 'rstcre [W m-2]', 'rsut [N/A]', 'rsutcs [N/A]', 'sfcWind [N/A]', 'ta-200 [N/A]', 'ta-850 [N/A]', 'tas [N/A]', 'tauu [N/A]', 'ts [N/A]', 'ua-200 [N/A]', 'ua-850 [N/A]', 'va-200 [N/A]', 'va-850 [N/A]', 'zg-500 [N/A]']\n",
       "len(var_list: 25\n",
-      "regions: ['NHEX_ocean', 'NHEX', 'SHEX_ocean', 'global', 'ocean', 'ocean_50S50N', 'NHEX_land', 'land', 'SHEX', 'TROPICS', 'SHEX_land']\n",
```

Comment:
The order of the regions was scrambled between notebooks, but the same regions are present in each of them.